SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1838,6 +1847,13 @@ void USpatialNetDriver::ProcessRPC(AActor* Actor, UObject* SubObject, UFunction*
 		Payload.PayloadData = RPCs->CreateRPCPayloadData(Function, Parameters);
 		FSpatialGDKSpanId SpanId = RPCs->CreatePushRPCEvent(CallingObject, Function);
 
+#if !UE_BUILD_SHIPPING
+		if (SpatialMetrics != nullptr)
```

Comment:
Just wonder if this should be being a flag?

Reply:
Looks at that, there is one - bEnableMetrics